robot/lib/resource.txt

```diff
@@ -0,0 +1,18 @@
+*** Settings ***
+Library           Collections
+Library           String
+Library           OperatingSystem
+Variables         ../data/variables.py
+
+*** Variables ***
+${OPENPOWER_HOST}        9.3.185.195
```

Comment:
Sooo.... if I don't specify a host, and I'm on the IBM network, I'll just run against your computer? You probably don't want that.